source/pages/address.tsx

```diff
@@ -59,7 +60,11 @@ if (environment.IS_CLIENT) {
   AddressPage = () => (
     <NoSSR>
       <Layout>
-        <Header withSearch brandType={BrandType.SHRINKED} />
+        <Header
+          withSearch
+          brandType={BrandType.SHRINKED}
+          searchProps={{ onSearch: noop }}
```

Comment:
This is causing an error when trying to search. It's better to return an empty function.

![image](https://user-images.githubusercontent.com/1504716/67034570-49a5dc00-f0ee-11e9-93b1-5e78482ef2fc.png)

Reply:
@daniloprates Oh, there was missing `{}` in noop imports.
I fixed them. Please review again.
Thanks.